grpcutil/dns_resolver.go

```diff
@@ -204,7 +206,7 @@ func (w *dnsWatcher) compileUpdate(newAddrs map[string]*Update) []*Update {
 
 func (w *dnsWatcher) lookupSRV() map[string]*Update {
 	newAddrs := make(map[string]*Update)
-	_, srvs, err := lookupSRV(w.ctx, "grpclb", "tcp", w.host)
+	_, srvs, err := lookupSRV(w.ctx, w.service, "tcp", w.host)
```

Comment:
Can we return early if `w.service == ""`?

Reply:
@pstibrany Makes sense. Implemented it.